splitio/client/client.go

```diff
@@ -59,17 +59,17 @@ func (c *SplitClient) getEvaluationResult(matchingKey string, bucketingKey *stri
 }
 
 // getEvaluationsResult calls evaluation for multiple treatments at once
-func (c *SplitClient) getEvaluationsResult(matchingKey string, bucketingKey *string, features []string, attributes map[string]interface{}, operation string) evaluator.Results {
+func (c *SplitClient) getEvaluationsResult(matchingKey string, bucketingKey *string, featureFlags []string, attributes map[string]interface{}, operation string) evaluator.Results {
 	if c.isReady() {
-		return c.evaluator.EvaluateFeatures(matchingKey, bucketingKey, features, attributes)
+		return c.evaluator.EvaluateFeatures(matchingKey, bucketingKey, featureFlags, attributes)
 	}
 	c.logger.Warning(operation + ": the SDK is not ready, results may be incorrect. Make sure to wait for SDK readiness before using this method")
 	c.initTelemetry.RecordNonReadyUsage()
 	result := evaluator.Results{
 		EvaluationTime: 0,
 		Evaluations:    make(map[string]evaluator.Result),
 	}
-	for _, feature := range features {
+	for _, feature := range featureFlags {
```

Comment:
Suggest to rename feature to featureFlag

Reply:
I updated. Thank you